packages/designmanual/stories/molecules/MyNdlaAddToFavoritesExample.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { MyNdlaFavoritesDialog } from '@ndla/ui';
+import { IconButton } from '@ndla/button';
+import { FavoriteHeart } from '@ndla/icons/action';
+
+const StyledFavoriteHeart = styled(FavoriteHeart)`
+  transform: scale(1.5);
```

Comment:
Her må width og height på ikonet endres heller enn å bruke transform. Det kan gjøres med font-size, men ikonets høyde og bredde må først endres (1em er default, men vi bruke .c-icon klassen til å sette 15px statisk)

```suggestion
  font-size: 1.5em; // Hva enn som passer. Her kan man også vurdere en prop "size" på IconButton for flere størrelser
  svg { // alternativt på .c-icon
    width: 1em
    height: 1em;
  }
```